Makefile.am

```diff
@@ -30,6 +24,6 @@ src/qos/sai_vm_queue.c         src/sai-db/sai_db_gen_utils.cpp \
 src/qos/sai_vm_sched_group.c   src/sai-db/sai_db_init.cpp \
 src/sai_vm_npu_api_query.c  src/sai_vm_npu_init_config.c  src/sai_vm_shell.c src/tunnel/sai_vm_tunnel.c
 
-libsai_npu_la_LDFLAGS = -lopx_logging -lopx_common -lsai-common-utils -lopx_db_sql
+libsai_npu_la_LDFLAGS = -zdefs -lopx_logging -lopx_common -lsai-common-utils -lopx_db_sql
```

Comment:
Hmm, -zdefs does essentially the same thing as dpkg-shlibdeps's underlinking checks.  There's no harm in having it here, but it's worth considering turning this on for all the opx repositories.